Data/FE15/Types/Person.yml

```diff
@@ -83,7 +83,10 @@ Person:
       type: message
       paths: [m/Name.bin.lz]
     - id: jid
```

Comment:
I can make the change after merging if you'd prefer, but maybe this should be changed to job? "jid" makes me think it's giving the actual JID string, not the record ID.